File: go/different.go
package main

import (
	"fmt"
)

func abs(num int64) int64 {
	if num < 0 {
		return num * -1
	}

	return num
}

func main() {

	var num1, num2 int64
	fmt.Scan(&num1, &num2)

	fmt.Println(abs(num2 - num1))

}

File: java/Different.java
import java.math.BigInteger;
import java.util.Scanner;

/**
 * https://open.kattis.com/problems/different/
 *
 * @author brunovolpato
 */
public class Different {
  public static void main(String[] args) {

    Scanner sc = new Scanner(System.in);

    while (sc.hasNextLine()) {
      String line = sc.nextLine();
      if (line.isEmpty()) {
        break;
      }

      String[] lineParts = line.split("\\s");
      long num1 = Long.valueOf(lineParts[0]);
      long num2 = Long.valueOf(lineParts[1]);

      System.out.println(Math.abs(num2 - num1));

    }
  }
}
